.github/workflows/test.yml

```diff
@@ -53,6 +53,10 @@ env:
   # To hide "Update available 0.0.0 -> x.x.x"
   PRISMA_HIDE_UPDATE_MESSAGE: true
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.event_name != 'schedule' }}
```

Comment:
https://docs.github.com/en/actions/using-jobs/using-concurrency#example-only-cancel-in-progress-jobs-or-runs-for-the-current-workflow

Reply:
`${{ github.event_name != 'schedule' }}` so that we don't cancel on scheduled runs (for BuildPulse)